deployment/terraform/modelservice/templates/application_yaml.tpl

```diff
@@ -38,3 +38,10 @@ graphite:
   urlTemplate: "${graphite_url}"
 security:
   signingKey: $${SIGNING_KEY}
+
+datasource-es:
+  create.index.if.not.found: true
```

Comment:
Spring uses periods for config hierarchy, which is not how you are using it here. Please replace this with either `createIndexIfNotFound` or `create-index-if-not-found`. (Spring uses both of those conventions for long key names and I don't care which we use.)

The `index.name` key below is fine because that makes sense as a hierarchical key.

Reply:
Updated.